main.js

```diff
@@ -18,7 +15,10 @@ require("yargonaut")
   .errorsStyle("red.bold");
 
 const yargs = require("yargs")
-  .demandCommand(1, chalk.red("[ERROR] 0 arguments passed. Please specify a command"))
+  .demandCommand(
+    1,
+    chalk.red("[ERROR] 0 arguments passed. Please specify a command")
+  )
```

Comment:
is it from `prettier` or from linter  ?

Reply:
I think this is from my own prettier. Lemme rollback

Reply:
cool

Reply:
done